aws-crt-sys/src/lib.rs

```diff
@@ -1,2 +1,945 @@
-// This is just a dummy file to make cargo realize this library needs to be built
-// It will be overwritten by the build process
\ No newline at end of file
+/* automatically generated by rust-bindgen 0.59.0 */
```

Comment:
not sure if it's cool or not that this is getting checked in.
I like being able to see it, but you say it can look different based on the platform that last generated it.
If it becomes annoying maybe we can add it to .gitignore so it gets ignored if it's different on the local machine?